src/pages/Admin/Venue/Rooms/RoomsForm.tsx

```diff
@@ -159,14 +159,14 @@ const RoomInnerForm: React.FC<RoomInnerFormProps> = (props) => {
   const [formError, setFormError] = useState(false);
 
   const onSubmit = useCallback(
-    async (vals: FormValues) => {
+    async (input: FormValues) => {
       if (!user) return;
 
       try {
         const roomValues: RoomInput = {
           ...editingRoom,
-          ...vals,
-          visibility: roomVisibility,
+          ...input,
+          visibility: input.visibility,
```

Comment:
why add it separately if we already have `...input` there?

Reply:
Didn't want to do the guesswork of the shape of all those inputs. There are like 5-6 different forms, so I opted to make it not overly presumptive. You can however update it upon merging to your branch as you see fit.